### PR TITLE
Fix mobile view of online members page with some additional UI/UX enhancements

### PR DESCRIPTION
**Pull request by ankitbajpai65**
**Issue** - https://github.com/Real-Dev-Squad/website-dashboard/issues/503

**Description** - 

1. Fixed the mobile view of online-members page. 

**Before** - https://github.com/Real-Dev-Squad/website-dashboard/issues/503

**Video after changes-**  

https://github.com/Real-Dev-Squad/website-dashboard/assets/85179394/f989186c-22ee-4b65-b53b-31b4cba12d05

2. Enable displaying a message "**No Results Found**" when user doesn't found any group.

**Before** - https://github.com/Real-Dev-Squad/website-dashboard/issues/503

**After** - 
image

3. Enhances UI of https://dashboard.realdevsquad.com/ in mobile view by keeping text of navbar to center and changing width of all boxes same.

**Before** - https://github.com/Real-Dev-Squad/website-dashboard/issues/503

**After -** 
image

4. Fixed the alignment issue. 

**Before** - https://github.com/Real-Dev-Squad/website-dashboard/issues/503

**After -** 
image
 
### Tests coverage
image
image

### CODE REVIEW

File: yarn.lock
Comment (Pratiyushkumar) [inline yarn.lock:1]: why are we importing yarn.lock file, any specific reason??

File: online-members/online-members.css [hunk 1]
```diff
@@ -123,3 +123,31 @@ footer .info-repo {
...
 }
+
+/* MEDIA QUERY */
+
+@media only screen and (max-width: 900px) {
+  .body {
+    flex-direction: column-reverse;
+    align-items: center;
+    padding-top: 2rem;
+    gap: 3rem;
```
Comment (Pratiyushkumar) [inline online-members/online-members.css:134]: why gap of 3rem is needed... any specific reason?? why not 10px or 5px??

**Review by Pratiyushkumar — CHANGES_REQUESTED** [review]
Please include test and test coverage

File: online-members/online-members.css (hunk 1 above)
Comment (ankitbajpai65) [reply online-members/online-members.css:134]: I have given spacing between two containers which are in column in mobile view. Spacing of 10px is not enough.

File: yarn.lock
Comment (ankitbajpai65) [reply yarn.lock:1]: @Pratiyushkumar Fixed this thing.

**Comment by Achintya-Chatterjee** [comment]
can you please add the test coverage?

**Comment by ankitbajpai65** [comment]
> Please include test and test coverage

@Pratiyushkumar Added the test coverage. Please look into this.

**Comment by ankitbajpai65** [comment]
> Added the test coverage. Please look into this.

@Achintya-Chatterjee Added the test coverage. Please look into this.

**Review by iamitprakash — APPROVED** [review]

**Review by Achintya-Chatterjee — DISMISSED** [review]

File: groups/script.js [hunk 2]
```diff
@@ -193,14 +193,23 @@ searchInput.addEventListener('keyup', () => {
...
+
+      if (displayValue === '') {
+        foundResults = true;
+      }
       loader.classList.add('hidden');
     });
+    const noResultsMessage = document.getElementById('no-results-message');
+    foundResults
+      ? (noResultsMessage.style.display = 'none')
+      : (noResultsMessage.style.display = 'block');
```
Comment (Aryex82) [inline groups/script.js:212]: Can we simplify this to something like this: 

```javascript
noResultsMessage.style.display = foundResults ? 'none' : 'block';
```

Comment (ankitbajpai65) [reply groups/script.js:212]: Yes, we can do that. Thanks for pointing @Aryex82

Comment (ankitbajpai65) [reply groups/script.js]: @Aryex82 I have done the changes. Please review the changes and approve.

**Review by Aryex82 — APPROVED** [review]

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**